mycpp/pass_state.py

```diff
@@ -215,9 +220,45 @@ def name(self) -> str:
         return 'assign'
 
     def Generate(self, func: str, statement: int) -> str:
-        return '{}\t{}\t{}\t{}\n'.format(func, statement,
-                                         SymbolPathToSouffle(self.lhs),
-                                         SymbolPathToSouffle(self.rhs))
+        return '{}\t{}\t{}\t$Ref({})\n'.format(
+            func, statement, SymbolPathToReference(func, self.lhs),
+            SymbolPathToReference(func, self.rhs))
+
+
+class Use(Fact):
+    """
+    The use of a reference.
```

Comment:
maybe give some examples of all kinds of code that generate this

Or a pointer to where that is documented